## File to prepare Data

In [1]:
#imports
import pandas as pd

### скачать файл из тг

In [2]:
data = pd.read_excel("../static/Book1.xls")
print(data)

     case №  age        BMI  gender  1test Ex  1test In  ComorbCCC  ComorbL  \
0         0   64  30.468750       0       103        61          1        0   
1         1   52  27.885187       0       110        58          1        0   
2         9   66  14.659928       0        79        14          1        0   
3        11   54  42.292640       1        69        19          0        0   
4        16   67  40.700000       0        58        32          1        0   
..      ...  ...        ...     ...       ...       ...        ...      ...   
379     442   81  20.775623       0        43        28          1        0   
380     443   66  29.996713       0        42        26          1        0   
381     444   57  26.397977       0        52        34          1        0   
382     445   63  25.109569       0        42        30          1        1   
383     446   67  26.297578       0        95        52          0        0   

     CD\Ozhir  ComorbAll  ...  chD  measurementday 

In [3]:
#delete some columns
data.drop('case №', axis=1, inplace=True)
data.drop('Result.1', axis=1, inplace=True)
data.drop('Unnamed: 22', axis=1, inplace=True)
data.drop('Unnamed: 23', axis=1, inplace=True)
data.drop('Unnamed: 24', axis=1, inplace=True)
print(data)

     age        BMI  gender  1test Ex  1test In  ComorbCCC  ComorbL  CD\Ozhir  \
0     64  30.468750       0       103        61          1        0         0   
1     52  27.885187       0       110        58          1        0         0   
2     66  14.659928       0        79        14          1        0         0   
3     54  42.292640       1        69        19          0        0         1   
4     67  40.700000       0        58        32          1        0         1   
..   ...        ...     ...       ...       ...        ...      ...       ...   
379   81  20.775623       0        43        28          1        0         0   
380   66  29.996713       0        42        26          1        0         0   
381   57  26.397977       0        52        34          1        0         1   
382   63  25.109569       0        42        30          1        1         1   
383   67  26.297578       0        95        52          0        0         0   

     ComorbAll  L 109     L

covid
|Параметр|Значение|Использование|
|-|-|-|
|case №|номер пациента в журнале|-|
|age|возраст|+|
|BMI|Индекс массы тела|+|
|gender|пол|0ж-1м|
|1test Ex|Экстубация|+|
|1test In|Интубация|+|
|ComorbCCC|Сердечн Сосуд заболеван|0-1|
|ComorbL|Заболев Лёгких|0-1|
|CD\Ozhir|ожирение|0-1|
|ComorbAll|ССС or L|0-1|
|L 109|лимфоциты|+|
|LF|другие лимфоциты|+|
|ROX|?|-|
|SpO2|насыщеность крови кислородом|+|
|SpO2\FiO|соотношения насыщения кислородом к фракции вдыхаемого кислорода (SpO (2) / FiO (2))|+|
|chD|чистота дыхания|+|
|measurementday|день первых измерений(каких-то)|-|
|dayshome|дни нахождения дома|+|
|Result|Результат(D - смэрт, H - здоров, R - реаним)|0-1-0|
|>7 or <7|> или < 7 дней в палате|0-1|
|daystoresult|точное время дней|+|

In [4]:
#save data
data.to_csv('data.csv', index=False)